ocs_ci/framework/testlib.py

```diff
@@ -2,7 +2,7 @@
 
 
 @pytest.mark.usefixtures(  # noqa: F405
-    'run_io_in_background'
+    'run_io_in_background', 'environment_checker'
```

Comment:
maybe switch?
To have the enc checker before you creating new pods?

Reply:
discussed